src/util.js

```diff
@@ -1,17 +1,12 @@
 // DOM properties that should NOT have "px" added when numeric
 export const IS_NON_DIMENSIONAL = /acit|ex(?:s|g|n|p|$)|rph|grid|ows|mnc|ntw|ine[ch]|zoo|^ord|^--/i;
 
-const HTML_ENTITY_REG = /[&<>"]/g;
-const tagsToReplace = {
-	'&': '&amp;',
-	'<': '&lt;',
-	'>': '&gt;',
-	'"': '&quot;'
-};
-const replaceTag = (tag) => tagsToReplace[tag] || tag;
 export function encodeEntities(s) {
-	if (typeof s !== 'string') s = String(s);
-	return s.replace(HTML_ENTITY_REG, replaceTag);
+	return String(s)
```

Comment:
Let's add a comment to summarize our findings from yesterdays quick benchmarking session.

Reply:
I've always felt that this change was suspicious and could be the root cause of the performance issues I reported previously for versions after 5.1.2.

I plan to deploy a test release next Monday and roll back this part of the change separately from the 5.1.16 version of the code to see if there are still performance issues.

Reply:
FYI
![image](https://user-images.githubusercontent.com/648674/112258726-f9ad3c00-8c34-11eb-92ce-6aede9ccd20a.png)

Reply:
@JimLiu - that benchmark website produces statistically meaningless results. It's effectively doing `while(performance.now()-start < 5000) test()`, which makes the order of benchmarks matter and doesn't reflect real JS VM workloads.

The only current benchmarking website that produces statistically meaningful results is [ESbench](https://esbench.com) (disclaimer: I maintain it). The reason for this is because it uses the same [Benchmark.js](https://benchmarkjs.com/) (the library that powered JSPerf, built by Matthias Bynens from the V8 team). Benchmark.js performs JIT warmup runs, encapsulates benchmarks in functions that prevent synthetic VM optimizations (like hoisting/inlining of benchmark code), and returns median values with standard deviation.

Here's the benchmarks we use for `encodeEntities` on ESBench:
https://esbench.com/bench/5f88af6cb4632100a7dcd414

<img width="615" alt="Screen Shot 2021-03-25 at 11 45 36 AM" src="https://user-images.githubusercontent.com/105127/112501458-a5eb4180-8d5f-11eb-854d-34a0e02c2b17.png">